ui/src/main/resources/edu/wpi/grip/ui/templates/cpp/Pipeline.h.vm

```diff
@@ -24,6 +24,16 @@ class $fileName{
 		#cType($source.type()) *$source.value();
 #end
 #foreach($step in $pipeline.getSteps())
+#set($boolNum = $velocityCount - 1)
+#if($step.name() == "Switch" || $step.name() == "Valve")
```

Comment:
Use `.equals` instead of `==`, there's no guarantee that `"Switch"` and "`Valve`" will be interned